recipes/chilife/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "chilife" %}
-{% set version = "1.0.0.dev6" %}
```

Comment:
Note that you can have the 1.0 dev version in a dev channel. The instructions should be in the CFEP-05 doc I linked in the previous comment. You can do that in the feedstock later if you wish.